Dockerfile

```diff
@@ -191,6 +202,11 @@ RUN find $ROOTFS/etc/rc.d/ $ROOTFS/usr/local/etc/init.d/ -exec chmod +x '{}' ';'
 # Change MOTD
 RUN mv $ROOTFS/usr/local/etc/motd $ROOTFS/etc/motd
 
+# Dirty hack to make the bootscript add an environment file to the .ssh/ directory
+RUN echo "touch /home/docker/.ssh/environment" >> $ROOTFS/bootscript.sh
+RUN echo "echo \"PATH=/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin\" >> /home/docker/.ssh/environment" >> $ROOTFS/bootscript.sh
+RUN echo "touch ./executed.log" >> $ROOTFS/bootscript.sh
```

Comment:
alternatively, we can change the boot2docker-cli tool to use the full path to the ip binary.